sloglint.go

```diff
@@ -438,3 +454,26 @@ func argsOnSameLine(fset *token.FileSet, call ast.Expr, keys, attrs []ast.Expr)
 
 	return false
 }
+
+func forbiddenKeysUsed(forbiddenKeys []string) func(string) bool {
+	return func(s string) bool {
+		// Strip quotes if any.
+		unquoted := trimQuotes(s)
+		return slices.Contains(forbiddenKeys, unquoted)
+	}
+}
+
+func valueChanged(handler func(string) string) func(string) bool {
```

Comment:
This looks complicated. Let's leave `badKeyNames` as is and implement the new option as simple `slices.Contains(forbiddenKeys, key)` looping over keys.

Reply:
@tigrato UPD. Nevermind, I forgot that we also need to check `Attr` keys. Let's just rename `caseFn` to something more general.

Reply:
Done in bce4e31c07313a64d973ac244d41fafb985751d2